Code/Tools/FBuild/FBuildTest/Tests/TestUnity.cpp

```diff
@@ -919,13 +919,21 @@ void TestUnity::CacheUsingRelativePaths() const
         }
 
         // Check __FILE__ paths are relative
-        // Slash direction changed in Clang 18.x.x from forward slash to backslash
-        TEST_ASSERT( buffer.Find( "FILE_MACRO_START_1(./Subdir/Header.h)FILE_MACRO_END_1" ) ||
-                     buffer.Find( "FILE_MACRO_START_1(.\\Subdir/Header.h)FILE_MACRO_END_1" ) );
 #if defined( __WINDOWS__ )
-        TEST_ASSERT( buffer.Find( "FILE_MACRO_START_2(.\\File.cpp)FILE_MACRO_END_2" ) );
+        TEST_ASSERTM(
+            buffer.Find( "FILE_MACRO_START_1(..\\out\\..\\Code\\Subdir/Header.h)FILE_MACRO_END_1" ),
+            "File content:\n%s",
+            buffer.Get() );
```

Comment:
The buffer is binary data, so I don't think we want to print out the output here.

I guess that was useful for tracking down the problem if you don't have access to all the environments, but I think it'd strange to print all manner of non-printable characters.